### PR TITLE
Upgrade the user guide and the UI labels

### PR DESCRIPTION
**Pull request by brandelune**
## Which ticket is resolved?

#1646 Update the user guide for version 5.8/6.0
https://sourceforge.net/p/omegat/feature-requests/1646/

#1647 Fix various UI labels for consistency
https://sourceforge.net/p/omegat/feature-requests/1647/

## What does this PR change?

- All the DocBook files
- The CSS file for the resulting HTML
- The Bundle.properties file
- Some .java files to update the UI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,14 @@ local.properties
...
+# emacs backup files
+*~
+
+# temporary 
+index.xml
+notes.org
+xhtml5/
+docs/
+/release/changes.org
+.gitignore
```
Comment (miurahr) [inline .gitignore:26]:
```suggestion

```

**Comment by miurahr** [comment]
Please don't update `docs` on same commit request.

**Comment by miurahr** [comment]
Please don't remove `releases/linux-specific`

**Comment by brandelune** [comment]
> Please don't update `docs` on same commit request.

> Please don't remove `releases/linux-specific`

Those are commit bugs on my side. Apologies.

**Comment by miurahr** [comment]
Changes are merged.

**Pull request closed without merging**